modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -636,6 +636,85 @@ definition {
 		}
 	}
 
+	@description = "LPS-178410 - Verify if actions works with oldValue function when trigger is On After Update (Notification) in First Name field"
+	@priority = 4
+	test CanCreateActionNotificationUsingOldValueWithOnAfterUpdateTriggerOnUserObject {
+		task ("Given: A notification template is created") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Notifications",
+				panel = "Control Panel",
+				portlet = "Templates");
+
+			ObjectNotifications.addNewNotificationTemplate(
+				entryTo = "test@liferay.com",
+				fromAddress = "test@liferay.com",
+				fromName = "Test Test",
+				name = "Email Template",
+				notificationType = "Email",
+				subject = "Subject Test");
+
+			PortletEntry.save();
+		}
+
+		task ("And given: A Notification action with the trigger (On After Update) is created and the oldValue function is used in the condition tab") {
```

Comment:
Can you add "for first name/givenName" to the end of this description
Something like "And given: A Notification action with trigger (On After Update) and oldValue function is used in the condition tab for first name/givenName"